bitnami/mongodb/templates/_helpers.tpl

```diff
@@ -15,6 +15,36 @@ If release name contains chart name it will be used as a full name.
 {{- include "common.names.fullname" . -}}
 {{- end -}}
 
+{{/*
+Create a default mongo service name which can be overridden.
+*/}}
+{{- define "mongodb.servicename" -}}
+    {{- if .Values.service -}}
+        {{- if .Values.service.name }}
+            {{- .Values.service.name -}}
+        {{- else -}}
+            {{ include "mongodb.fullname" . }}-headless
+        {{- end -}}
+    {{- else -}}
+        {{ include "mongodb.fullname" . }}-headless
+    {{- end }}
+{{- end }}
+
+{{/*
+Create a default mongo arbiter service name which can be overridden.
+*/}}
+{{- define "mongodb.arbiterservicename" -}}
+    {{- if .Values.arbiter.service -}}
+        {{- if .Values.arbiter.service.name }}
```

Comment:
Change servicename to service.nameOverride

Reply:
Fixed